frontend/package.json

```diff
@@ -20,7 +22,11 @@
     "start": "react-scripts start --ignore client",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src",
+    "lint:fix": "eslint src --fix",
+    "prettier": "prettier src",
+    "prettier:check": "prettier src --check"
```

Comment:
I would like the option of having a pre-commit hook (not necessarily in this PR, but this reminded me), are any of these are "stable" enough that it would be safe to automatically run them before 99.9% of commits?

Reply:
Lint:fix should be pretty stable from my experience, it only changes the styles and doesn't edit any of the code. And we can even use lint to check for errors using the CI as it returns all styling, syntax and other errors.

Reply:
Good to know, I'll take a look at adding this as a pre-commit step later